x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/config/AnalysisConfig.java

```diff
@@ -164,8 +160,6 @@ public AnalysisConfig(StreamInput in) throws IOException {
                 }
             }
         }
-
-        usePerPartitionNormalization = in.readBoolean();
```

Comment:
Here we need to check that if we are reading from an older node we consume the boolean (although we do nothing with it).